packages/framework/fluid-static/src/types.ts

```diff
@@ -14,7 +14,7 @@ export type LoadableObjectRecord = Record<string, IFluidLoadable>;
 
 /**
  * A mapping of string identifiers to classes that will later be used to instantiate a corresponding `DataObject`
- * or `SharedObject` in a {@link LoadableObjectRecord}.
```

Comment:
A public item shouldn't link to an internal one - this is semantically inappropriate, and the generated documentation will not be able to resolve the link.

This is a scenario for which we would ideally be able to add detection in the future, but for now detection is somewhat manual.